### PR TITLE
feat(rgbpp-sdk/ckb): Add examples with queue service

### PR DESCRIPTION
**Pull request by duanyytop**
## Main Changes

- Add examples with queue service 
- Update the CKB Tx fee of the jumping transaction from CKB to BTC
- Add failedReason to the RGBPP state response
- Update examples readme

### CODE REVIEW

File: examples/rgbpp/README.md [hunk 1]
```diff
@@ -25,6 +25,44 @@ But when you're searching for this transaction in [Bitcoin Core](https://bitcoin
...
+```
+
+### Unlock BTC time cells on CKB
+
+**Warning: Wait at least 6 BTC confirmation blocks to unlock the BTC time cells after 4-btc-jump-ckb.ts**
+
+```shell
+npx ts-node examples/rgbpp/queue/5-spend-btc-time-cell.ts 
+```
+
```
Comment (Flouse) [inline examples/rgbpp/README.md:63]: I think the step of unlocking btc-time-lock cell is automatically handled by the RGB++ Quere service.

Comment (duanyytop) [reply examples/rgbpp/README.md:63]: YES. It's an error description and I will remove it

File: examples/rgbpp/README.md [hunk 2]
```diff
@@ -25,6 +25,36 @@ But when you're searching for this transaction in [Bitcoin Core](https://bitcoin
...
+```shell
+npx ts-node examples/rgbpp/queue/3-btc-transfer.ts 
+```
+
+### Jump RGB++ asset from BTC to CKB
+
+```shell
+npx ts-node examples/rgbpp/queue/4-btc-jump-ckb.ts 
+```
+
```
Comment (Flouse) [inline examples/rgbpp/README.md:55]:
```suggestion

### Unlock BTC time cells on CKB
A cron job in RGB++ queue service will construct a transaction unlocking the mature btc_time_lock cells to the their `target_ckb_address`.
```

**Comment by Dawn-githup** [comment]
- [x] Reuse the Omiga protocol to quickly issue XUDT assets on CKB
- ckb address:  ckt1qyqztkx6kxv8p6esy6w7cm65lgd858z8elts5nqa6q
- Xudt has been minted with tx hash 0x33b84348f8ed87db48c9530d975a4fc69bed4fcb4a0287bf6a82acdaa9ad5dc2
- [x] Jump XUDT from CKB to BTC
```
- BTC UTXO TxId: 
- 210b564b2eed01278093f017b32c12737ece1125d924ac850821f7822a2f084e
```
- ckb address:  ckt1qyqztkx6kxv8p6esy6w7cm65lgd858z8elts5nqa6q
- Rgbpp asset has been jumped from CKB to BTC and tx hash is 0x557d462e0f10592807d9521cedfe1f513a69345ab054838d4a0007f682504470

- [x] Transfer RGBPP asset on BTC
```
- BTC UTXO TxId: 
e61fab10c391e331c37f31aae6591c0fca225795549cbefd43052132fcab06e1
```
```
- BTC TxId:  210b564b2eed01278093f017b32c12737ece1125d924ac850821f7822a2f084e
- to btc address:  tb1qm06rvrq8jyyckzc5v709u7qpthel9j4d9f7nh3
```
BTC TxId:  25bb12f93eeeaaf7c7657075924c4cd2450def1b66c72e669f0469d49e5445b2

- [x] Jump RGBPP asset from BTC to CKB
```
- BTC TxId:  25bb12f93eeeaaf7c7657075924c4cd2450def1b66c72e669f0469d49e5445b2
```
ckb address:  ckt1qyqztkx6kxv8p6esy6w7cm65lgd858z8elts5nqa6q
btc address:  tb1qm06rvrq8jyyckzc5v709u7qpthel9j4d9f7nh3
BTC TxId:  54245a7579e4e36ec49b392a9e9c70d2abb7b3ddc8ed3d864610bc4e2afa6a99

File: examples/rgbpp/queue/3-btc-transfer.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  try {
+    await service.sendRgbppCkbTransaction({ btc_txid: btcTxId, ckb_virtual_result: ckbVirtualTxResult });
+    const interval = setInterval(async () => {
+      const { state, failedReason } = await service.getRgbppTransactionState(btcTxId);
+      console.log('state', state);
+      if (state === 'completed' || state === 'failed') {
+        clearInterval(interval)
+        if (state === 'completed') {
+          const { txhash: txHash } = await service.getRgbppTransactionHash(btcTxId);
+          console.info(`Rgbpp asset has been transferred on BTC and tx hash and tx hash is ${txHash}`);
```
Comment (Flouse) [inline examples/rgbpp/queue/3-btc-transfer.ts:88]:
```suggestion
          console.info(`Rgbpp asset has been transferred on BTC and the related CKB tx hash is ${txHash}`);
```

File: examples/rgbpp/queue/4-btc-jump-ckb.ts [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  try {
+    await service.sendRgbppCkbTransaction({ btc_txid: btcTxId, ckb_virtual_result: ckbVirtualTxResult });
+    const interval = setInterval(async () => {
+      const { state, failedReason } = await service.getRgbppTransactionState(btcTxId);
+      console.log('state', state);
+      if (state === 'completed' || state === 'failed') {
+        clearInterval(interval);
+        if (state === 'completed') {
+          const { txhash: txHash } = await service.getRgbppTransactionHash(btcTxId);
+          console.info(`Rgbpp asset has been jumped from BTC to CKB and tx hash is ${txHash}`);
```
Comment (Flouse) [inline examples/rgbpp/queue/4-btc-jump-ckb.ts:93]:
```suggestion
          console.info(`Rgbpp asset has been jumped from BTC to CKB and the related CKB tx hash is ${txHash}`);
```

**Review by Flouse — APPROVED** [review]

**Pull request merged**